app/src/main/java/chat/rocket/android/chatroom/adapter/MessageViewHolder.kt

```diff
@@ -57,7 +57,10 @@ class MessageViewHolder(
             button_join_video_call.setOnClickListener { joinVideoCallListener(it) }
 
             image_avatar.setImageURI(data.avatar)
-            text_content.setTextColor(if (data.isTemporary) Color.GRAY else Color.BLACK)
+            if (data.isTemporary){
+                text_content.setTextColor(Color.GRAY)
```

Comment:
This way the text color will be always gray. What about the data is not temporary?

Reply:
If it isnt temporary, it uses the color defined in the xml. However, I have to change this to use colors other than gray depending on the theme.